aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -316,6 +316,11 @@ void persistGlueTable(
               .skipArchive(awsProperties.glueCatalogSkipArchive())
               .tableInput(
                   TableInput.builder()
+                      // Keep the existing table description
+                      .description(glueTable.description())
+                      // This overwrites the existing table description if there's a comment like
```

Comment:
> This overwrites the existing table description

You mean `applyMutation` overwrites right? I'd just explicitly say that on the comment in 319

```
// Explicitly keep the existing table description since `applyMutation` will clear the table description
```

Reply:
> You mean applyMutation overwrites right?

Yes.

> // Explicitly keep the existing table description since `applyMutation` will clear the table description

This is not right. `applyMutation` won't clear the table description, it overwrites the description only if the comment is specified by the user as below:

```
 ALTER TABLE prod.db.sample SET TBLPROPERTIES ('comment' = 'A table description.')
```

I want to explain why I need to call `.description(glueTable.description())` first and call `applyMutation` second. If the order is reversed, user cannot update the table description by using the comment.

Reply:
Updated the comment in the latest change